src/pyhf/cli/rootio.py

```diff
@@ -69,14 +79,15 @@ def json2xml(workspace, output_dir, specroot, dataroot, resultprefix, patch):
         )
     from .. import writexml
 
+    _ensure_dirs(output_dir)
```

Comment:
@lukasheinrich This shouldn't have been added as this was removed during the Python 3 only migration.